en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -176,28 +379,32 @@ class MyPage extends Page
     private static $has_many = [
         'Banners' => Banner::class
     ];
+
     private static $owns = [
         'Banners'
     ];
 }
+
 class Banner extends DataObject
 {
     private static $extensions = [
         Versioned::class
     ];
+
     private static $has_one = [
         'Parent' => MyPage::class,
         'Image' => Image::class,
     ];
+
     private static $owns = [
         'Image'
     ];
 }
 ```
 
-If a `MyPage` gets published, all its related `Banners` will also be published, which will cause all `Image` DataObjects to be published.
+If a `MyPage` record gets published, all its related `Banners` will also be published, which will cause all related `Image` records to be published.
```

Comment:
Will $owns  also cascade unpublish/archive as well? If so should mention that here.

Reply:
Nope - `cascade_deletes` is used for cascading those operations. I'll mention that here.